tcti/tcti_socket.c

```diff
@@ -191,14 +191,22 @@ tcti_socket_cancel (
     TSS2_TCTI_CONTEXT_INTEL *tcti_intel = tcti_context_intel_cast (tctiContext);
     TSS2_RC rc;
 
+    /* the TCTI must have executed the transmit function successfully */
     rc = tcti_common_checks (tctiContext);
     if (rc != TSS2_RC_SUCCESS) {
         return rc;
-    } else if (tcti_intel->status.commandSent != 1) {
+    }
+    if (tcti_intel->state != TCTI_STATE_RECEIVE) {
```

Comment:
It look like now the status.commandSent is only used in setLocality and we can use the tcti_intel->state and remove the status.commandSent.

Reply:
I wrote another series today that removes all of the `status.*` stuff. Will make another PR once this goes in.